scripts/blocks-utils.js

```diff
@@ -160,7 +171,8 @@ function decorateQuickLinks(main) {
       block.id = toCamelCase(quickLinkTitle);
     }
   };
-  main.querySelectorAll('div.section-container > div > div').forEach(addQuickLinksMetadata);
+  main.querySelectorAll('div.tabs-container[data-quicklinks-title]').forEach(handQuickLinksMetadataForTabs);
+  main.querySelectorAll('div.section-container > div > div').forEach(addQuickLinksMetadataForBlocks);
```

Comment:
The section may have multiple blocks, can we handle this case?

Reply:
@JiangLong2019 I am handling both cases: 
1. For all blocks in the page, I am looking for data-quicklinks-title present on any block, it will look for all blocks anywhere on the page. 
2. For any section on the page where there is section-level data-quicklinks-title I am processing that as well.

Reply:
Looks good